alpine-chroot-install

```diff
@@ -345,6 +362,9 @@ printf '%s\n' \
 dump_alpine_keys etc/apk/keys/
 
 cp /etc/resolv.conf etc/resolv.conf
+if [ "x$FORCE_DNS" != "x" ]; then
+	sed -i "/^nameserver/s/ .*$/ $FORCE_DNS/" etc/resolv.conf
```

Comment:
There may be (and typically are) multiple `nameserver`s in `resolv.conf`, so this will result in multiple `nameserver` records with the same address. It shouldn’t break anything, but it’s not correct. Moreover, why not simple write only the specified DNS(s) to `resolv.conf`instead of modifying the one copied from the host?